spartan/aztec-network/files/config/genesis.json

```diff
@@ -0,0 +1,179 @@
+{
+    "config": {
+      "chainId": 1337,
+      "homesteadBlock":0,
+      "eip150Block":0,
+      "eip155Block":0,
+      "eip158Block":0,
+      "byzantiumBlock":0,
+      "constantinopleBlock":0,
+      "petersburgBlock":0,
+      "istanbulBlock":0,
+      "muirGlacierBlock":0,
+      "berlinBlock":0,
+      "londonBlock":0,
+      "arrowGlacierBlock":0,
+      "grayGlacierBlock":0,
+      "mergeNetsplitBlock":0,
+      "bedrockBlock":0,
+      "regolithTime":0,
+      "shanghaiTime":0,
+      "cancunTime":0,
+      "terminalTotalDifficulty":0,
+      "terminalTotalDifficultyPassed":true
+    },
+    "nonce": "0x42",
+    "timestamp": "0x0",
+    "extraData": "0x11bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82fa",
+    "gasLimit": "0x1388",
+    "difficulty": "0x400000000",
+    "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "coinbase": "0x0000000000000000000000000000000000000000",
+    "alloc": {
+      "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266": {
```

Comment:
100 eth to our default test accounts, can easily extend this to add whatever accounts we want